File: LJZY.MODEL/LQ_SCDT.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace LJZY.MODEL
{
    public class LQ_SCDT
    {
        public LQ_SCDT()
        {
            _XQXMB = "";
            _SGDH = "";
            _JDDT = "";
        }
        private int _TROW;
        /// <summary>
        /// 排序序号
        /// </summary>
        [DisplayName ( "TROW" )]
        public int TROW
        {
            get { return _TROW; }
            set { _TROW = value; }
        }

        private string _ZJH;
        /// <summary>
        /// 井号
        /// </summary>
        [DisplayName ( "ZJH" )]
        public string ZJH
        {
            get { return _ZJH; }
            set { _ZJH = value; }
        }

        private string _LJFGS;
        /// <summary>
        /// 录井项目部
        /// </summary>
        [DisplayName ( "LJFGS" )]
        public string LJFGS
        {
            get { return _LJFGS; }
            set { _LJFGS = value; }
        }

        private string _XQXMB;
        /// <summary>
        ///辖区项目部
        /// </summary>
        [DisplayName ( "XQXMB" )]
        public string XQXMB
        {
            get { return _XQXMB; }
            set { _XQXMB = value; }
        }

        private string _DWZBH;
        /// <summary>
        /// 队伍自编号
        /// </summary>
        [DisplayName ( "DWZBH" )]
        public string DWZBH
        {
            get { return _DWZBH; }
            set { _DWZBH = value; }
        }


        private string _LJYQXH;
        /// <summary>
        /// 设备型号
        /// </summary>
        [DisplayName ( "LJYQXH" )]
        public string LJYQXH
        {
            get { return _LJYQXH; }
            set { _LJYQXH = value; }
        }

        private string _LJDH;
        /// <summary>
        /// 录井队号
        /// </summary>
        [DisplayName ( "LJDH" )]
        public string LJDH
        {
            get { return _LJDH; }
            set { _LJDH = value; }
        }

        private string _SGDH;
        /// <summary>
        /// 施工队号
        /// </summary>
        [DisplayName("SGDH")]
        public string SGDH
        {
            get { return _SGDH; }
            set { _SGDH = value; }
        }

        private string _JDDT;
        /// <summary>
        /// 井队动态
        /// </summary>
        [DisplayName("JDDT")]
        public string JDDT
        {
            get { return _JDDT; }
            set { _JDDT = value; }
        }

        private string _XDT;
        /// <summary>
        /// 井动态
        /// </summary>
        [DisplayName ( "XDT" )]
        public string XDT
        {
            get { return _XDT; }
            set { _XDT = value; }
        }

        private decimal _SJJS;
        /// <summary>
        /// 设计井深
        /// </summary>
        [DisplayName ( "SJJS" )]
        public decimal SJJS
        {
            get { return _SJJS; }
            set { _SJJS = value; }
        }

        private string _DRJS;
        /// <summary>
        /// 当日井深
        /// </summary>
        [DisplayName ( "DRJS" )]
        public string DRJS
        {
            get { return _DRJS; }
            set { _DRJS = value; }
        }

        private string _KZRQ;
        /// <summary>
        /// 开钻日期
        /// </summary>
        [DisplayName ( "KZRQ" )]
        public string KZRQ
        {
            get { return _KZRQ; }
            set { _KZRQ = value; }
        }

        private string _YJWJRQ;
        /// <summary>
        /// 预计完井日期
        /// </summary>
        [DisplayName ( "YJWJRQ" )]
        public string YJWJRQ
        {
            get { return _YJWJRQ; }
            set { _YJWJRQ = value; }
        }

        private string _HXJW;
        /// <summary>
        /// 后续井位
        /// </summary>
        [DisplayName ( "HXJW" )]
        public string HXJW
        {
            get { return _HXJW; }
            set { _HXJW = value; }
        }

        private string _HXJDH;
        /// <summary>
        /// 后续井队号
        /// </summary>
        [DisplayName ( "HXJDH" )]
        public string HXJDH
        {
            get { return _HXJDH; }
            set { _HXJDH = value; }
        }

        private string _SGZT;
        /// <summary>
        /// 井队动态
        /// </summary>
        [DisplayName ( "SGZT" )]
        public string SGZT
        {
            get { return _SGZT; }
            set { _SGZT = value; }
        }

        private string _REPORT_TYPE;
        /// <summary>
        /// 井别
        /// </summary>
        [DisplayName ("REPORT_TYPE")]
        public string REPORT_TYPE
        {
            get { return _REPORT_TYPE; }
            set { _REPORT_TYPE = value; }
        }

        private string _LSDW;
        /// <summary>
        /// 隶属单位
        /// </summary>
        [DisplayName ( "LSDW" )]
        public string LSDW
        {
            get { return _LSDW; }
            set { _LSDW = value; }
        }

        private string _DQZT;
        /// <summary>
        /// 当前状态
        /// </summary>
        [DisplayName ( "DQZT" )]
        public string DQZT
        {
            get { return _DQZT; }
            set { _DQZT = value; }
        }

        private string _BZ;
        /// <summary>
        /// 备注
        /// </summary>
        [DisplayName ( "BZ" )]
        public string BZ
        {
            get { return _BZ; }
            set { _BZ = value; }
        }
    }
}

File: LJZY.WEB/Scripts/LQZY/LQ_SCDT.js
var app = new Vue({
    el: "#SCDT",
    data: {
        tips: false,
        position: 'absolute',
        top: 0,
        left: 0,
        XMList: [],
        lineHeight: 40,
        XMBList: [],
        XMBName: '',
        LJFGS: '',
        changeData: '',
        changeTime: '',
        changeTIME: '',
        SGDHList: '',
        
    },
    
//    var _this = this;           
//$.ajax({
//    type: "POST",
//    url: "../../Controllers/SCDTController.ashx?action=SCDT_List",
//    data: { LJFGS: _this.XMBName, Time: $('#datecheckbox').datebox('getValue') },
//    dataType: "json",
//    success: function (msg) {
//        // console.log(msg)
//        _this.changeTime = $('#datecheckbox').datebox('getValue');
//        //console.log(msg, "日期：" + $('#datecheckbox').datebox('getValue'))   
//        //console.log(msg);

//        _this.XMList = msg;
//        if (msg.data !== 'undefined' && msg.data !== null && msg.data !== '') {
//            var dataList = '';
//            dataList = { "code": msg.code, "count": msg.count, "msg": msg.msg, "data": msg.data }
//        }
//        layui.use('table', function () {
//            var table = layui.table;

//            table.render({
//                elem: '#test'
//                , data: dataList.data
//                , page: false
//                , limit: 999999
//                , width: $(document.body).width() - 100 + 'px'
//                , height: $(document.body).height() - 200 + 'px'
//                , cols: [[
//                      { field: 'XQXMB', width: 120, title: '辖区项目部', fixed: 'left' }
//                    , { field: 'TROW', width: 100, title: '序号', fixed: 'left' }
//                    , { field: 'DWZBH', width: 100, title: '编号', fixed: 'left' }
//                    , { field: 'LJYQXH', width: 120, title: '仪器' }
//                    , { field: 'ZJH', width: 100, title: '井号' }
//                    , { field: 'LJDH', width: 100, title: '录井队号' }
//                    , { field: 'SGDH', width: 100, title: '施工队号' }
//                    , { field: 'XDT', width: 100, title: '井动态' }
//                    , { field: 'SJJS', width: 100, title: '设计井深' }
//                    , { field: 'DRJS', width: 100, title: '当日井深', }
//                    , { field: 'KZRQ', width: 100, title: '开钻日期' }
//                    , { field: 'YJWJRQ', width: 100, title: '预计完井日期' }
//                    , { field: 'HXJW', width: 120, title: '录井后续井位' }
//                    , { field: 'HXJDH', width: 100, title: '录井后续钻井队号' }
//                    , { field: 'JDDT', width: 100, title: '钻井队动态' }
//                    , { field: 'HXJW', width: 120, title: '钻井队后续井' }
//                    , { field: 'JB', width: 100, title: '井别' }
//                    , { field: 'LJFGS', width: 100, title: '隶属' }
//                    , { field: 'DQZT', width: 100, title: '状态' }
//                    , { field: 'BZ', width: 100, title: '备注', fixed: 'right' }
//                ]]
                           
                           
//            });
//        });

//    }
//})
    methods: {
        getData: function () {
            var _this = this;
            var SELECT_XMB = $("#SELECT_XMB option:selected");
            console.log(SELECT_XMB.text())

            $.ajax({
                type: "POST",
                url: "../../Controllers/SCDTController.ashx?action=SCDT_List",
                data: { LJFGS: _this.XMBName, Time: $('#datecheckbox').datebox('getValue') },
                dataType: "json",
                success: function (msg) {
                     console.log(msg)
                            _this.changeTime = $('#datecheckbox').datebox('getValue');
                            //console.log(msg, "日期：" + $('#datecheckbox').datebox('getValue'))   
                            //console.log(msg);

                            _this.XMList = msg;
                            if (msg.data !== 'undefined' && msg.data !== null && msg.data !== '') {
                                var dataList = '';
                                dataList = { "code": msg.code, "count": msg.count, "msg": msg.msg, "data": msg.data }
                            }
                            layui.use('table', function () {
                                var table = layui.table;

                                table.render({
                                    elem: '#test'
                                    , data: dataList.data
                                    , page: false
                                    , limit: 999999
                                    , width: $(document.body).width() - 100 + 'px'
                                    , height: $(document.body).height() - 200 + 'px'
                                    , cols: [[
                                        { field: 'XQXMB', width: 120, title: '辖区项目部', fixed: 'left', align: 'center' }
                                        , { field: 'TROW', width: 100, title: '序号', fixed: 'left', align: 'center'}
                                        , { field: 'DWZBH', width: 100, title: '编号', fixed: 'left', align: 'center'}
                                        , { field: 'LJYQXH', width: 120, title: '仪器', align: 'center'}
                                        , { field: 'ZJH', width: 100, title: '井号', align: 'center' }
                                        , { field: 'LJDH', width: 100, title: '录井队号', align: 'center' }
                                        , { field: 'SGDH', width: 100, title: '施工队号', align: 'center' }
                                        , { field: 'XDT', width: 100, title: '井动态', align: 'center'}
                                        , { field: 'SJJS', width: 100, title: '设计井深', align: 'center'}
                                        , { field: 'DRJS', width: 100, title: '当日井深', align: 'center' }
                                        , { field: 'KZRQ', width: 100, title: '开钻日期', align: 'center'}
                                        , { field: 'YJWJRQ', width: 120, title: '预计完井日期', align: 'center'}
                                        , { field: 'HXJW', width: 120, title: '录井后续井位', align: 'center' }
                                        , { field: 'HXJDH', width: 150, title: '录井后续钻井队号', align: 'center' }
                                        , { field: 'JDDT', width: 120, title: '钻井队动态', align: 'center'}
                                        , { field: 'HXJW', width: 120, title: '钻井队后续井', align: 'center' }
                                        , { field: 'REPORT_TYPE', width: 100, title: '井别', align: 'center'}
                                        , { field: 'LJFGS', width: 100, title: '隶属', align: 'center'}
                                        , { field: 'DQZT', width: 100, title: '状态', align: 'center' }
                                        , { field: 'BZ', width: 100, title: '备注', fixed: 'right', align: 'center' }
                                    ]]


                                });
                            });
                }
            })
        },
        getXMB: function () {
            _this = this;
            $.ajax({
                type: "POST",
                url: "../../Controllers/ColumnListController.ashx?action=List_LJFGS",
                dataType: "json",
                success: function (msg) {
                    console.log(msg);
                    _this.XMBList = msg;
                }
            })
        },


    },
    created: function () {
        var that = this; 
        that.getXMB();
    },

})



//导出
function exportExcel() {
    //console.log($("#ff"))
    //console.log($('#SELECT_XMB option:selected')[0].value, $('#SELECT_XMB option:selected'))
    $('#ExportIframe').attr("src", "../../../Page/DownLoad/DownLoad.aspx?Source=OutExportExcel_SCDT& LJFGS=" + $('#SELECT_XMB option:selected').val() + "&Time=" + $('#datecheckbox').datebox('getValue'));
}
